R/opts.R

```diff
@@ -538,6 +548,7 @@ rstan_sampling_opts <- function(cores = getOption("mc.cores", 1L),
                                 future = FALSE,
                                 max_execution_time = Inf,
                                 ...) {
+  dot_args <- list(...)
```

Comment:
better code style?

Reply:
I catch it here for modification later in the function because `iter` has two values in the final list if it is passed through `...` and also recalculated below.